code/modules/research/designs/stock_parts_designs.dm

```diff
@@ -306,4 +306,4 @@
 	build_type = PROTOLATHE
 	materials = list(MAT_GLASS = 100, MAT_SILVER = 10, MAT_URANIUM = 15)
```

Comment:
But it works for the protolathe why not just add it to autolathe no problem in material costs